src/riak_dt.proto

```diff
@@ -22,7 +22,8 @@
 */
 
 /*
-** Revision: 2.0
+%% TODO: Set for whichever release version we use
+** Revision: 2.3.0
```

Comment:
I left this question to @JeetKunDoug w/ the release time (we didn't know at the time)... so @nickelization, you're good to do that (and keep it as a separate commit) if Riak team is ok w/ it ;).